xds/src/main/java/io/grpc/xds/internal/sds/SdsProtocolNegotiators.java

```diff
@@ -245,12 +253,21 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause)
 
   private static final class ServerSdsProtocolNegotiator implements ProtocolNegotiator {
 
-    // TODO (sanjaypujare) integrate with xDS client to get LDS. LDS watcher will
-    // inject/update the downstreamTlsContext from LDS
     private DownstreamTlsContext downstreamTlsContext;
+    private final XdsClientWrapperForServerSds xdsClientWrapperForServerSds;
 
-    ServerSdsProtocolNegotiator(DownstreamTlsContext downstreamTlsContext) {
+    ServerSdsProtocolNegotiator(
+        DownstreamTlsContext downstreamTlsContext, int port, SynchronizationContext syncContext) {
       this.downstreamTlsContext = downstreamTlsContext;
+      XdsClientWrapperForServerSds localXdsClientWrapperForServerSds;
+      try {
+        localXdsClientWrapperForServerSds =
+            XdsClientWrapperForServerSds.newInstance(port, Bootstrapper.getInstance(), syncContext);
+      } catch (Exception e) {
+        logger.log(Level.SEVERE, "XdsClientWrapperForServerSds constructor", e);
```

Comment:
nit: rephrase the error message. This is above-DEBUG-level logging, so it should be semantically meaningful to readers. It should be something like "Failed to ...". Also, "SEVER" means something is very broken and the system is probably panic. But it turns out we are still able to proceed by falling back to plaintext. I think if something required is missing and we have nothing to do, we just let it panic and halt. But if we do have alternative to recover, it may just be a warning.

Same thing may apply for other log messages.